client/src/modules/patients/record/patient-record.html

```diff
@@ -83,6 +83,13 @@
                     <span class="glyphicon glyphicon-upload"></span> {{ PatientRecordCtrl.uploadButtonText | translate }}
                     <input type="file" accept="image/*" ngf-max-size="2MB" ngf-select="PatientRecordCtrl.uploadFiles($file, $invalidFiles)" style="display: none;">
                 </label>
+                <a
+                  href
+                  ng-click="PatientRecordCtrl.openWebcam()"
+                  class="btn btn-default btn-block">
+                  <span class="glyphicon glyphicon-edit"></span>
+                  <span translate>WebCam</span>
```

Comment:
This should conform to normal translated text.  It should also say something like "Take a Picture".

Reply:
you're right